include/envoy/api/os_sys_calls.h

```diff
@@ -12,6 +12,10 @@
 namespace Envoy {
 namespace Api {
 
+struct EnvoyTcpInfo {
+  uint32_t tcpi_rtt; // Round Trip Time (RTT) in microseconds.
```

Comment:
You could potentially make it std::chrono::microseconds type, or add a suffix to indicate units to minimize conversion problems. Strongly typing is probably better.

Reply:
fixed in a4c473a91d2bcc3bdc15ce56f6a7dcc7bbad7556. I added  `std::chrono::microseconds` in the struct to be consistent with the unix struct. But I maintained the backwards compatibility in the api because I assume that would break the consumers of the function outside of envoy